phac-nml/irida-next
# [ENHC0010032] List samples within transfer dialog

## What does this PR do and why?
Added a scrollable list within the sample transfer dialog to display the selected samples.
Fixes #468.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/3e91d217-5801-4d9e-a3d0-598931c6663d)
![image](https://github.com/phac-nml/irida-next/assets/325703/00ec628b-3250-4663-8414-87a49462a1d3)

## How to set up and validate locally
1. Open a rails console and run the following:
 
```
project = Project.first
 
11.upto(10000) do |i|
  sample = Samples::CreateService.new(
    project.creator, project,
    { name: "#{project.namespace.parent.name}/#{project.name} Sample #{i}",
      description: "This is a description for sample #{project.namespace.parent.name}/#{project.name} Sample #{i}." 
    }
  ).execute
end
```
2. Login as admin and navigate to [http://localhost:3000/bacillus/bacillus-anthracis/outbreak-2022/-/samples](http://localhost:3000/bacillus/bacillus-anthracis/outbreak-2022/-/samples).
3. Select all the samples within the table.
4. Click the `Transfer samples` button.
5. Verify the selected samples are displayed.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
